Tests/test_elasticsearch.py

```diff
@@ -0,0 +1,148 @@
+# ================================================================================================
+# Tests for Checking the Elasticsearch Layer
+# ================================================================================================
+
+import os
+import json
+import yaml
+import logging
+import argparse
+import requests
+
+
+def init(cluster_name):
+    # Initialize logging module
+    logging.basicConfig(level=logging.INFO, format='%(asctime)s - [%(levelname)s] - %(message)s', datefmt='%Y-%m-%d %H:%M:%S')
+    logging.info("The cluster name is: " + cluster_name)
+
+    # Record Elasticsearch and Kibana Endpoint
+    ELB_ENDPOINT    = 'https://' + os.popen('kubectl get svc -o json | jq \'.items[] | select(.metadata.name == "' + cluster_name + '-es-http") | .status.loadBalancer.ingress[0].hostname\'').read().rstrip().strip('"') + ':9200'
```

Comment:
Not too important, but you can easily select one svc by running:
`kubectl get svc/cluster_name-es-http`

Reply:
Thanks, I overlooked this.

Reply:
Commit added for the same.